### ISSUE TITLE
inference_warning

### ISSUE THREAD

**Issue by llllly26** [Alpha-VLLM/Lumina-mGPT#34]
Hi, thanks for your work!

when I run the simple inference code, the shell shows that: "

> Some weights of ChameleonForConditionalGeneration were not initialized from the model checkpoint ..., You should probably TRAIN this model on a down-stream task to be able to use it for predictions and inference.  

and

> The attention mask and the pad token id were not set. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.Setting `pad_token_id` to `eos_token_id`:8710 for open-end generation.The attention mask is not set and cannot be inferred from input because pad token is same as eos token. As a consequence, you may observe unexpected behavior. Please pass your input's `attention_mask` to obtain reliable results.

How to solve it? thanks!

**Comment by KZF-kzf** [comment]
i meet the same problem, have you solved this?
